<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="8,3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,6">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="connectBtn">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tabControl">
           <attribute name="title">
            <string>Control</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9" rowstretch="5,0">
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Single measurement</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Parameters</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Initial angle</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBox">
                 <property name="minimum">
                  <number>-90</number>
                 </property>
                 <property name="maximum">
                  <number>90</number>
                 </property>
                 <property name="value">
                  <number>-45</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Final angle</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBox_2">
                 <property name="minimum">
                  <number>-90</number>
                 </property>
                 <property name="maximum">
                  <number>90</number>
                 </property>
                 <property name="value">
                  <number>45</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Step</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="doubleSpinBox">
                 <property name="decimals">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <double>5.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QPushButton" name="updateAngleBtn">
                 <property name="text">
                  <string>Update</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="2">
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Signal threshold</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QSpinBox" name="signalThresholdValueBox">
                 <property name="maximum">
                  <number>2400</number>
                 </property>
                 <property name="value">
                  <number>1600</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabScan">
           <attribute name="title">
            <string>Scan</string>
           </attribute>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Scan type</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QRadioButton" name="rBtn_TOF_scan">
                 <property name="text">
                  <string>TOF only</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rBtn_PAS_scan">
                 <property name="text">
                  <string>PAS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="singleMeasurementBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Single</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="scanBtn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Continuous</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>Measurement</string>
              </property>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Left</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLCDNumber" name="lcd_left">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                   <strikeout>false</strikeout>
                   <stylestrategy>PreferDefault</stylestrategy>
                   <kerning>true</kerning>
                  </font>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="midLineWidth">
                  <number>0</number>
                 </property>
                 <property name="smallDecimalPoint">
                  <bool>false</bool>
                 </property>
                 <property name="digitCount">
                  <number>5</number>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Right</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLCDNumber" name="lcd_right">
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>Active sensor</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QCheckBox" name="cBox_left_sensor">
                 <property name="text">
                  <string>Left sonar</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cBox_right_sensor">
                 <property name="text">
                  <string>Right sonar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QGroupBox" name="groupBox_8">
              <property name="title">
               <string>Algorithm</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QCheckBox" name="cBox_alg1">
                 <property name="text">
                  <string>Alg1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Sonar map</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QCustomPlot" name="sonarMap" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>2</horstretch>
               <verstretch>3</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Analog voltage envelope</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCustomPlot" name="sonarSignal" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSerial_port"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to the uC</string>
   </property>
  </action>
  <action name="actionSerial_port">
   <property name="text">
    <string>Serial port</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
